msal-web-sample/src/main/java/com/microsoft/azure/msalwebsample/AuthPageController.java

```diff
@@ -43,55 +39,47 @@ public ModelAndView securePage(HttpServletRequest httpRequest) throws ParseExcep
         ModelAndView mav = new ModelAndView("auth_page");
 
         setAccountInfo(mav, httpRequest);
-
         return mav;
     }
 
     @RequestMapping("/msal4jsample/sign_out")
-    public void signOut(HttpServletRequest httpRequest, HttpServletResponse response) throws ParseException, IOException {
-
+    public void signOut(HttpServletRequest httpRequest, HttpServletResponse response) throws IOException {
```

Comment:
should we also invalidate cache entry here ?

Reply:
I believe line 47 here invalidates the session, which includes the cache entry.